src/test/java/de/tum/in/www1/artemis/util/HestiaUtilTestService.java

```diff
@@ -109,7 +109,7 @@ public ProgrammingExercise setupTemplate(Map<String, String> files, ProgrammingE
                 eq(false), any());
 
         bitbucketRequestMockProvider.enableMockingOfRequests(true);
-        bitbucketRequestMockProvider.mockDefaultBranch("master", urlService.getProjectKeyFromRepositoryUrl(templateRepoUrl));
+        bitbucketRequestMockProvider.mockDefaultBranch("main", urlService.getProjectKeyFromRepositoryUrl(templateRepoUrl));
```

Comment:
Can you use the value defined in the yml file? You can inject it easily

Reply:
Done